### PR TITLE
use xhtml friendly spaces

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes https://github.com/PolymerElements/iron-autogrow-textarea/issues/47

### CODE REVIEW

**Comment by jmillikin** [comment]
I notice the diff contains only one line; the <div id="mirror"> on line 81 also references &nbsp; and needs to be changed.

**Comment by notwaldorf** [comment]
Done!

**Comment by jmillikin** [comment]
Thank you very much, LGTM!

File: iron-autogrow-textarea.html [hunk 1]
```diff
@@ -317,7 +318,8 @@
       while (this.rows > 0 && _tokens.length < this.rows) {
         _tokens.push('');
       }
-      return _tokens.join('<br>') + '&nbsp;';
+      // Use &#160; instead &nbsp; of to allow this element to be used in XHTML.
+      return _tokens.join('<br/>') + '&#160;';
```
Comment (cdata) [inline iron-autogrow-textarea.html]: :dragon_face:

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**